search-index/index_data.py

```diff
@@ -222,20 +234,25 @@ def run_cli(
         main_dataset, ids_table, embs, description_embs_dict
     )
 
+    def _convert_to_bool(x):
```

Comment:
Small helper because OpenSearchIndex needs bools, not the strings parsed from environment variables.

Reply:
A small thing but some code in the backend (see `backend.app.core.config`) which does a similar thing is case-insensitive. Can we make `x.lower() == "true"/"false"` here instead?

Reply:
done